src/main/java/com/mitchej123/hodgepodge/mixins/Mixins.java

```diff
@@ -9,6 +9,7 @@
 
 public enum Mixins {
     // Vanilla Fixes
+    FIX_SAW_ICE_BREAK("minecraft.MixinBlockIce", () -> true, TargetedMod.VANILLA),
```

Comment:
Please make this mixin optional, like all others

Reply:
IT fixes a water dupe bug, why make it optional ?

Reply:
Because A) it is good practice and B) this will throw an ClassNotFoundException if GregTech is not installed

Reply:
People outside of the pack use hodgepodge, and I'll likely be pushing it to CF/Modrinth. All patches need to be configurable. This patch should also depend on GT, not vanilla.

Reply:
should be good